scripts.js

```diff
@@ -0,0 +1,21 @@
+var span = $("span");
+    
+span.each(function(index, element) {
+	if(index % 2 == 0) {
+		$(element).css('color', 'red');
+	};
+});
+
+var paragraphs = $('p');
+paragraphs.each(function(index, element) {
+	
+
+	var button = '<button class="btn" data-tmp="' + index + '">Click me</button>'
+	$(element).append(button)
+});
+
+$("button").click(function(){
+	alert($(this).attr("data-tmp"));
+});
+
+
```

Comment:
remove last empty line - on the end of document you should leave only one empty line